### ISSUE TITLE
the negative power number be formatted to subtraction when assignment of module attributes

### ISSUE THREAD

**Issue by Ljzn** [elixir-lang/elixir#13402]
### Elixir and Erlang/OTP versions

Elixir 1.16.1 (compiled with Erlang/OTP 26)

### Operating system

MacOS

### Current behavior

```ex
@a -(2**2)
```

run `mix format`, the code be formatted to:

```ex
@a - 2 ** 2
```

### Expected behavior

```ex
@a -(2 ** 2)
```

**Comment by Ljzn** [comment]
interesting thing is, if we write like:
```ex
@a (-(2**2))
```

will be formatted to:
```ex
@a -(2 ** 2)
```

Looks great. But what if we format it again...

**Comment by josevalim** [comment]
Thanks for the report. I have a fix for it but it can only be included on Elixir v1.17.0 because it is larger than i expected. Meanwhile, I recommend writing it as:

```
val = -(2**2)
@a val
```

Sorry :(
